crates/schematic/Cargo.toml

```diff
@@ -134,6 +134,7 @@ reqwest = { workspace = true, features = [
 serial_test = "3.2.0"
 similar = "2.7.0"
 starbase_sandbox = { workspace = true }
+derive_more = { version = "2.0.1", features = ["try_into", "as_ref"] }
```

Comment:
I unfortunately had to add a `dev-dependency` here for the tests because I can't think of any macros in the standard library that could be used to test this.